### PR TITLE
Create language conventions

### LINKED ISSUE #5: Language conventions
Create language conventions

### PR DESCRIPTION
**Pull request by Mathias-a**
Closes #5

### CODE REVIEW

File: docs/conventions/language_conventions.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Language Conventions
+
+The entire code-base, branches, commits and issues should all be written in english.
+
+Pull Requests should be written in norwegian.
```
Comment (emilte) [inline docs/conventions/language_conventions.md:5]: Kanskje `allows both Norwegian and English`

File: docs/conventions/language_conventions.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+When in doubt, default to writing in english.
```
Comment (emilte) [inline docs/conventions/language_conventions.md:7]: Nice!

**Review by emilte — APPROVED** [review]

File: docs/conventions/language_conventions.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Language Conventions
+
+The entire code-base, branches, commits and issues should all be written in english.
```
Comment (emilte) [inline docs/conventions/language_conventions.md:3]: test

File: docs/conventions/language_conventions.md (hunk 1 above)
Comment (Mathias-a) [reply docs/conventions/language_conventions.md]: Enig, fixed

**Pull request merged**